### PR TITLE
fix: Revert change of extension which should refer to the new file extension

### PR DESCRIPTION
**Pull request by juliushaertl**
Signed-off-by: Julius Härtl <jus@bitgrid.net>

* Resolves: # 
* Target version: main

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] Sign-off message is added to all commits
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: lib/TemplateManager.php [hunk 1]
```diff
@@ -478,7 +478,7 @@ public function formatNodeReturn(File $template) {
 			'preview' => $this->urlGenerator->linkToRouteAbsolute('richdocuments.templates.getPreview', ['fileId' => $template->getId()]),
 			'type' => $this->flipTypes()[$template->getMimeType()],
 			'delete' => $this->urlGenerator->linkToRouteAbsolute('richdocuments.templates.delete', ['fileId' => $template->getId()]),
-			'extension' => $template->getExtension(),
+			'extension' => $ooxml ? self::TYPE_EXTENSION_OOXML[$documentType] : self::TYPE_EXTENTION[$documentType],
```
[the remark this answers is not in the document]
Comment (juliushaertl) [reply lib/TemplateManager.php]: Good catch, fixed

**Review by luka-nextcloud — APPROVED** [review]

**Pull request merged**